lib/galaxy/datatypes/constructive_solid_geometry.py

```diff
@@ -868,3 +871,142 @@ def sniff_prefix(self, file_prefix: FilePrefix) -> bool:
         False
         """
         return self._has_root_element_in_prefix(file_prefix, "VTKFile")
+
+
+@build_sniff_from_prefix
+class Vtp:
+    """
+    A VTP file is a Visualization Toolkit (VTK) file format that specifically stores polygonal data
+    (surface meshes) in a hierarchical, XML-based format. It's designed to efficiently represent and
+    communicate 3D geometric models with associated data attributes.
+    """
+
+    subtype = ""
+
+    # Add metadata elements (sorted alphabetically by name).
+    MetadataElement(name="file_format", default=None, desc="File format", readonly=True, optional=True, visible=True)
```

Comment:
What are those file_formats? Should we have subclasses of this for each of the file_format? E.g. can there be tools or Vis that require one of the specific sub-types?

Is unusual to encode file_formats this way I think.

Reply:
`file_format` here refers to `type` in the VTP-file, commonly its `PolyData`. I used `file_format` because it is used in `VTK` and `PLY` but I switched it to `type` so it aligns with actual attribute name in the VTP-file.

Reply:
I followed up one more time, and remove file_format/type, instead the blurb is now showing any of the other metadata attributes collected from the dataset e.g. 
<img width="289" alt="image" src="https://github.com/user-attachments/assets/18e035ba-d318-4282-aae0-f2f1363d8dfd" />